lib/auth.js

```diff
@@ -231,6 +234,47 @@ module.exports = new Model({
     }.bind(this));
   },
 
+  signInWithOAuthRedirect: function (args, window) {
```

Comment:
Ideally we'd be able to run browser-specific things in a browser, but I'm not sure that's worth investigating right now (I have no idea how we'd run a test across different windows and origins). In PFE we're using a `window-shim` lib to render on the server. Might be overkill here.